lib/jekyll/site_data/file.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module Jekyll
+  module SiteData
+    class File
+      attr_accessor :context
+      attr_reader :content
+
+      def initialize(site, path, content)
```

Comment:
This needs a method comment describing the inputs, since it's a bit vague (e.g. is "path" relative or absolute?)